math/R/Portfile

```diff
@@ -200,6 +200,12 @@ platform darwin 13 {
 
 variant accelerate conflicts atlas builtin_lapack openblas description {build using the BLAS and Lapack in Apple's Accelerate framework} {
     configure.args-append   --with-blas="-framework Accelerate" --with-lapack
+    if { ([vercmp ${xcodeversion} >= 14.3] || \
+         [vercmp ${xcodecltversion} >= 14.3]) && \
+         (${os.platform} eq "darwin" && ${os.version} >=  22.3) } {
+       configure.args-append --with-newAccelerate=lapack
+       configure.args-delete --with-blas="-framework Accelerate" --with-lapack
```

Comment:
Please change this so that appropriate args are added if conditions are met, not added and then deleted if conditions are met. Other than this LGTM.

Reply:
changed the relevant part and force pushed it.